addon/webextension/background/main.js

```diff
@@ -325,5 +325,11 @@ this.main = (function() {
     }));
   });
 
+  // Used for the preview images. Can be removed once #3508
+  // (https://git.io/v5Nkc) is done.
+  communication.register("dataUrlToBlob", (sender, url) => {
```

Comment:
Let's just include blobConverters in content worker and call this function directly instead of using the background worker